project/plugins.sbt

```diff
@@ -11,9 +11,9 @@ ThisBuild / libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml"
 
```

Comment:
`"HMRC-open-artefacts-ivy"` can be changed to `"HMRC-open-artefacts-ivy2"`

`resolvers += Resolver.typesafeRepo("releases")` this line is no longer needed